pages/common/busybox.md

```diff
@@ -3,7 +3,7 @@
 > A collection of small system utilities in a single executable.
 > Executing `busybox` via a symlink is equivalent to running `busybox symlink_name`.
 > Linux distributions that use BusyBox will usually provide symlinks for all programs.
-> More information: <https://www.busybox.net/downloads/BusyBox.html>.
+> More information: <https://busybox.net>.
```

Comment:
```suggestion
> More information: <https://www.busybox.net/downloads/BusyBox.html>.
```
This was better as it was. The original page directed the users directly to usage information.